guides/cookbook.md

```diff
@@ -1061,6 +1186,101 @@ class ReadNumericValues
 ```
 </div>
 
+<div class="tabpane Go" markdown="1">
+
+A reader can have various types of numeric values:
+  - Int32
+  - Int64
+  - Uint64
+  - BigInt
+  - Float
+  - Decimal
+
+The following example illustrates how to read different numeric values in a reader:
+```Go
+package main
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/amzn/ion-go/ion"
+)
+
+func main() {
+	int32Value := 2147483646
+	var int64Value int64 = 9223372036854775807
```

Comment:
```suggestion
	 int64Value int64 := 9223372036854775807
```

Not sure why we need `var` only for this line and not the others

Reply:
I needed to explicitly define a value as int64 so I can test ```reader.Int64Value()```.
Assignment like ```int64Value := 9223372036854775808```, will infer ```value64``` as type of ```Int```.

For consistency, changed the definition here and explicitly cast it to ```int64``` on line 1241.